src/input/mod.rs

```diff
@@ -1461,6 +1477,30 @@ impl State {
                     }
                 }
             }
+            Action::MoveWorkspaceToMonitor(new_output) => {
```

Comment:
These two need to warp pointer, but only when the new output is different from the current workspace output. Might be convenient to return a bool from the layout on whether the workspace moved to a new output.